helm/ako/templates/statefulset.yaml

```diff
@@ -214,6 +214,11 @@ spec:
               configMapKeyRef:
                 name: avi-k8s-config
                 key: autoFQDN
+          - name: IP_SCHEME
+            valueFrom:
+              configMapKeyRef:
+                name: avi-k8s-config
+                key: ipscheme
```

Comment:
key must be **ipScheme** instead, right?

Reply:
Thanks corrected.